packages/plugin/src/utils.ts

```diff
@@ -99,3 +99,14 @@ export function extractTokens(source: string): AST.Token[] {
 
   return tokens;
 }
+
+export function checkForEslint(token: Token, rawNode: DocumentNode): boolean {
+  if (token.kind !== 'Comment') {
+    return false;
+  }
+  const string = rawNode.loc.source.body.substring(token.start+1, token.start+8);
```

Comment:
@gilgardosh the `loc` might be optional. Can you please add optional chaining here?